data/src/main/java/com/mument_android/data/repository/SignRepositoryImpl.kt

```diff
@@ -32,4 +35,10 @@ class SignRepositoryImpl @Inject constructor(
         }
     }
 
+    override suspend fun getViewView(page: String): WebViewEntity? {
+        signDataSource.getWebView(page).let {
+            return getWebViewMapper.map(it.data)
+        }
+    }
+
```

Comment:
- 여기서 it에 대한 Null check를 아래와 같이 하는게 나을 것 같습니다~!
```suggestion
    override suspend fun getViewView(page: String): WebViewEntity? {
        signDataSource.getWebView(page)?.let {
            return getWebViewMapper.map(it.data)
        }
    }

```

Reply:
오호 이해했습니다!! 감사합니다~!